e2e/pageobjects/ide/Ide.ts

```diff
@@ -123,22 +135,42 @@ export class Ide {
     async waitStatusBarTextAbcence(expectedText: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
         const statusBarLocator: By = By.css('div[id=\'theia-statusBar\']');
 
-        await this.driverHelper.getDriver().wait(async () => {
-            const elementText: string = await this.driverHelper.waitAndGetText(statusBarLocator, timeout);
+        // for ensuring that check is not invoked in the gap of status displaying
+        for (let i: number = 0; i < 3; i++) {
```

Comment:
I don't really get this *for* cycle... It seems to me you just want to increase timeout for the `wait`. So why not just increase the timeout?